package.json

```diff
@@ -129,6 +130,7 @@
     "json-rpc-engine": "^5.3.0",
     "json-rpc-middleware-stream": "^2.1.1",
     "jsonschema": "^1.2.4",
+    "lavamoat": "^4.1.1",
```

Comment:
dep should actually be `lavamoat-core` , thats where we are importing ses from

Reply:
changed